HedgeLib/Sets/SetObjectType.cs

```diff
@@ -68,11 +69,18 @@ public override void Load(Stream fileStream)
             {
                 string elemName = element.Name.LocalName;
                 var typeAttr = element.Attribute("type");
+                RawLength = 0;
                 if (typeAttr == null) continue;
 
                 if (elemName.ToLower() == "extra")
                 {
                     // TODO: Parse Extra XML elements.
+
+                    // Length of raw parameter data
+                    if (typeAttr.Value == "RawBitLength")
```

Comment:
Really nit-picky thing here, but this doesn't seem to actually represent the length of all of the object's parameters in bits as the name suggests.

Rather, it represents the length of all of the object's parameters in bytes.
Again this is a really minor detail, but yeah I think the name is currently a little misleading. "RawByteLength" would be more accurate.